modules/User.php

```diff
@@ -1,6 +1,108 @@
 <?php
 	class User {
-		function __construct() {
-			return;
+		public $email;
+		public function __construct($email) {
+			if (Utils::isValidEmail($email)) {
+				$this->email = $email;
+			} else {
+				return NULL;
+			}
+		}
```

Comment:
I do not think returning NULL is a good idea. Can we change it to something else?